### PR TITLE
Prevent Illegal reflective access in JAUtilities

### PR DESCRIPTION
**Pull request by jansupol**
Signed-off-by: jansupol <jan.supol@oracle.com>

### CODE REVIEW

**Comment by jansupol** [comment]
Fixes #555

**Comment by pzygielo** [comment]
> Seems the build on Jenkins just hangs waiting on executor where there is no available executor at all for HK2. What can I do?

Submit bug for instance to be restarted.

File: pom.xml [hunk 1]
```diff
@@ -150,7 +150,7 @@
         <glassfish.javax.el.version>3.0.1-b10</glassfish.javax.el.version>
         <jakarta.el.version>3.0.2</jakarta.el.version>
         <jtype.version>0.1.0</jtype.version>
-        <javassist.version>3.22.0-CR2</javassist.version>
+        <javassist.version>3.27.0-GA</javassist.version>
```
[the remark this answers is not in the document]
Comment (jansupol) [reply pom.xml:153]: it just needs to adjust the policy file

**Comment by arjantijms** [comment]
@dmatej can you check again? The tests pass now.

**Review by dmatej — APPROVED** [review]

**Pull request merged**